google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ViewDefinition.java

```diff
@@ -201,7 +201,11 @@ Table toPb() {
       viewDefinition.setUserDefinedFunctionResources(Lists.transform(userDefinedFunctions,
           UserDefinedFunction.TO_PB_FUNCTION));
     }
-    viewDefinition.setUseLegacySql(useLegacySql);
+    if (useLegacySql == null) {
```

Comment:
This changes the interpretation of `null`. After this change on Apiary level `null` for a flag named `useLegacySql` will be still interpreted as "use legacy SQL", but on veneer level it will be "use standard SQL". This is a type of confusion which can be and should be avoided (as it may lead to stupid bugs in the future). 

Please strongly consider just having `private Boolean useLegacySql = false;` and keeping everything else as is. This should allow having standard SQL by default, but if somebody wants to change it, the behavior will be consistent with the rest.